### PR TITLE
Automated-Gateways 

### PR DESCRIPTION
**Pull request by Koosha-E-Khorasani**
Automated Gateways is a secure solution which can make Contract-Call interoperatibility happen between Hyperledger Fabrics with minimal efforts.

### CODE REVIEW

File: labs/automated_gateways.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Automated-Gateways
+parent: Labs
+---
+
+# Lab Name
+ Automated-Gateways
+# Short Description
+Automated Gateways is a secure solution which can make Contract-Call interoperatibility happen between Hyperledger Fabrics with minimal efforts.
```
Comment (tkuhrt) [inline labs/automated_gateways.md:10]:
```suggestion
Automated Gateways is a secure solution which can make Contract-Call interoperability happen between Hyperledger Fabric networks with minimal efforts. 
```

File: labs/automated_gateways.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+parent: Labs
+---
+
+# Lab Name
+ Automated-Gateways
+# Short Description
+Automated Gateways is a secure solution which can make Contract-Call interoperatibility happen between Hyperledger Fabrics with minimal efforts. 
+
+# Scope of Lab
+By utilizing three smart contracts as the foundation of authentication and authorization, the Automated Gateways module provides a simple and secure method for Hyperledger fabrics to exchange services and data. When a user integrates this module into their code and deploys three predetermined smart contracts, they can grant access to their services to other Hyperledger fabrics. Moreover, they can access services from smart contracts deployed on other Hyperledger fabrics. Through this method, various Hyperledger fabrics can connect and utilize each other's services.
```
Comment (tkuhrt) [inline labs/automated_gateways.md:13]:
```suggestion
By utilizing three smart contracts as the foundation of authentication and authorization, the Automated Gateways module provides a simple and secure method for Hyperledger Fabric networks to exchange services and data. When a user integrates this module into their code and deploys three predetermined smart contracts, they can grant access to their services to other Hyperledger Fabric networks. Moreover, they can access services from smart contracts deployed on other Hyperledger Fabric networks. Through this method, various Hyperledger Fabric networks can connect and utilize each other's services.
```

File: labs/automated_gateways.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+# Sponsor
+N/A
+
+** This project is developed and maintained by [TCDT Lab](https://tcdt.ca/) of the University of Manitoba
+
+# Pre-existing repository
+
+- https://github.com/tcdt-lab/Automated-Gateways
```
Comment (tkuhrt) [inline labs/automated_gateways.md:38]: It looks like the commits in this repository do not contain DCO signoff. Hyperledger requires that all contributed code has a DCO sign off. Looking at these repositories, it appears that this is not the case. There are two options that we normally consider for fixing this:

1. Amend existing commits in the repository to include DCO sign off
1. Manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: labs/automated_gateways.md (hunk 3 above)
Comment (Koosha-E-Khorasani) [reply labs/automated_gateways.md:38]: I though having my name and email as Author (shown after git log) would cover it. Sorry, my bad!
I will take care of it.
screenshot

Comment (Koosha-E-Khorasani) [reply labs/automated_gateways.md:38]: It is resolved @tkuhrt .

**Review by tkuhrt — APPROVED** [review]

**Review by arsulegai — APPROVED** [review]
LGTM

**Pull request merged**